end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -2959,7 +2985,7 @@ importance of proper experimental planning and the need to control for all
 potential confounding factors. For the present experiment, no disease-specific
 biomarker could be identified, but a life-style property of individuals
 suffering from this disease: coffee consumption should be reduced for this
-particular disease to reduce the risk of heart arrhythmias.
+particular disease to reduce the risk of heart arrhythmias.(But CVD patients had lower abundance of this ion then when compared with control group: FT0845 had a fold change of -6.334. Since caffeine was decreased on the disease group, doesn`t seem that our data tells us that the disease is related to caffeine consumption (Although it does make sense biologically). Also is though to make good conclusion from such analysis, so I would replace this alst sentence by something like "but a probable life-style property that separate disease group from controls")
```

Comment:
Thanks for catching this, I think this is very badly explained on my side. What we want to highlight is that cardiovascular patients were probably advised to reduce caffeine consumption and therefore resulting in the control group having much higher caffeine presence. I will rephrase.